Onboarding/Sources/Onboarding/Views/OnboardingContainerView.swift

```diff
@@ -18,7 +18,9 @@ struct OnboardingContainerView: View {
             VStack(spacing: 0) {
                 TabView(selection: $viewModel.tabIndex) {
                     ForEach(0..<onboardingSlides.count, id: \.self) { index in
-                        OnboardingSlideView(model: onboardingSlides[index])
+                        OnboardingSlideView(model: onboardingSlides[index]).onAppear {
+                            viewModel.trackNavigationEvent()
```

Comment:
Are you going to double-track here? Won't this fire when the continue button is pressed?